datasets/doc/source/index.rst

```diff
@@ -7,6 +7,15 @@ learning/analytics/evaluation. It is created by the ``Flower Labs`` team that al
 Flower Datasets Framework
 -------------------------
 
+Install
+~~~~~~~
+
+.. code-block:: bash
+
+  python -m pip install flwr-datasets[vision]
```

Comment:
```suggestion
  python -m pip install "flwr-datasets[vision]"
```

Reply:
this change is needed for `zsh` shells. We should probably also update the installation how-to page.